manageability-docker/TBH/inbm-third-party-programs.txt

```diff
@@ -0,0 +1,1794 @@
+Turtle Creek Third Party Program Files
```

Comment:
We should be using the third-party-programs.txt file only.  We are only using one file now for the product as a whole instead of 2 separate since the Open source.  https://github.com/intel/intel-inb-manageability/blob/develop/third-party-programs.txt